tests/functional/conftest.py

```diff
@@ -19,20 +19,18 @@ def event_loop():
 
 
 @pytest.fixture(scope='session')
-def selenoid_container():
-    with BrowserWebDriverContainer(settings.test_caps) as firefox:
+def firefox_container():
+    with BrowserWebDriverContainer(settings.Firefox.CONTAINER) as firefox:
```

Comment:
Migrate to DockerContainer and use the following images:
```
CHROME("browsers/chrome:104.0”),
    EDGE("browsers/edge:104.0”),
    FIREFOX("browsers/firefox:103.0”),
    OPERA("browsers/opera:89.0”),
    SAFARI("browsers/safari:15.0");
```

Reply:
fixed in 9b20364a666f03413286df31a3d4c2cfb919f0c5